WebSocket connection error on prerelease2.triplea-game.org

**How can the problem be recreated?**
- Open TripleA using latest master
- Click on "Play Online" and login with your credentials
- Get the error: "Failed to connect to: wss://prerelease2.triplea-game.org/lobby/games/listing-ws"

Not sure what the exact implications are, but the dialog is annoying ^^

**When the problem is fixed, what should have happened instead?**
Just the usual connect to lobby stuff

**Which Engine Version are you using?** lastest master

**Screenshots**
![Screenshot](https://user-images.githubusercontent.com/8350879/75779146-666b4500-5d59-11ea-8018-40296396331d.png)

@DanVanAtta I actually don't think it's an TLS problem.

I got this stacktrace:
```
SEVERE: Websocket error
javax.net.ssl.SSLException: Socket closed
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:127)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:320)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:263)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:258)
	at java.base/sun.security.ssl.SSLSocketImpl.handleException(SSLSocketImpl.java:1313)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:839)
	at java.base/java.io.InputStream.read(InputStream.java:205)
	at org.java_websocket.client.WebSocketClient.run(WebSocketClient.java:424)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.net.SocketException: Socket closed
	at java.base/java.net.SocketInputStream.socketRead0(Native Method)
	at java.base/java.net.SocketInputStream.socketRead(SocketInputStream.java:115)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:168)
	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:140)
	at java.base/sun.security.ssl.SSLSocketInputRecord.read(SSLSocketInputRecord.java:448)
	at java.base/sun.security.ssl.SSLSocketInputRecord.bytesInCompletePacket(SSLSocketInputRecord.java:68)
	at java.base/sun.security.ssl.SSLSocketImpl.readApplicationRecord(SSLSocketImpl.java:1103)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:823)
	... 3 more
```

Which indicates that the server decided to close the connection at some point, not some sort of TLS problems.

Because local works, I believe it must be an nginx configuration issue that is is somehow not letting the websocket request through and just blocking it.

--- a/infrastructure/ansible/roles/nginx/templates/etc_nginx_sites_enabled_default.j2
+++ b/infrastructure/ansible/roles/nginx/templates/etc_nginx_sites_enabled_default.j2
@@ -53,5 +53,11 @@ server {
       proxy_set_header Upgrade $http_upgrade;
       proxy_set_header Connection "upgrade";
     }
-}
 
+    location /lobby/games/listing-ws {
+      proxy_pass http://localhost:8080;
+      proxy_http_version 1.1;
+      proxy_set_header Upgrade $http_upgrade;
+      proxy_set_header Connection "upgrade";
+    }
+}

